crates/wash-cli/Cargo.toml

```diff
@@ -63,7 +63,7 @@ wadm = { workspace = true }
 warp = { workspace = true }
 wascap = { workspace = true }
 wash-lib = { workspace = true, features = ["cli", "parser", "nats", "start"] }
-wasmcloud-control-interface = { workspace = true }
+wasmcloud-control-interface = { version = "1.0.0-alpha.1" }
```

Comment:
Note that this crate is pinned here and in wash-lib until I cut the new host release